client/me/help/help-contact-form/index.jsx

```diff
@@ -94,6 +95,18 @@ module.exports = React.createClass( {
 		this.setState( { siteSlug } );
 	},
 
+	trackClickStats: function( selectionName, option ) {
+		if ( selectionName === 'howCanWeHelp' ) {
+			analytics.tracks.recordEvent( 'calypso_help_how_can_we_help_click', {
+				selected_option: option.value
+			} );
+		} else if ( selectionName === 'howYouFeel' ) {
+			analytics.tracks.recordEvent( 'calypso_help_how_you_feel_click', {
+				selected_option: option.value
+			} );
+		}
+	},
+
```

Comment:
Just a style thing, but this could also be written with a switch statement:

```js
switch ( selectionName ) {
    case 'howCanWeHelp':
        return analytics.tracks.recordEvent( 'calypso_help_how_can_we_help_click', {
            selected_option: option.value
        } );
    case 'howYouFeel':
        return analytics.tracks.recordEvent( 'calypso_help_how_you_feel_click', {
            selected_option: option.value
        } );
}
```

Or, easier to extend if we add more options later:

```js
const tracksEvent = {
    howCanWeHelp: 'calypso_help_how_can_we_help_click',
    howYouFeel: 'calypso_help_how_you_feel_click'
}[ selectionName ];

if ( tracksEvent ) {
    analytics.tracks.recordEvent( tracksEvent, { selected_option: option.value } );
}
```